app/src/main/java/org/xwiki/android/sync/contactdb/SyncTypeGroupsList.kt

```diff
@@ -0,0 +1,12 @@
+package org.xwiki.android.sync.contactdb
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import org.xwiki.android.sync.bean.XWikiGroup
+
+@Entity(tableName = "groups_list")
```

Comment:
Table name in all tables must be extracted in const val and reused in annotations, which are related to this table

Reply:
Done.